### ISSUE TITLE
Adjust global font size

### ISSUE THREAD

**Issue by nicodh** [deltachat/deltachat-desktop#890]
It would be nice (and concerning accessibility more than nice) to be able to adjust the font size.
Maybe this can be done together with the #889 Theming?

**Comment by Simon-Laux** [comment]
We should do like 3-5 presets that we know don't break the layout

**Comment by Simon-Laux** [comment]
maybe `.bp3-large`can help a bit?

**Comment by Simon-Laux** [comment]
we could use the css unit `rem` for this (which is relative to font-size of the root element)

**Comment by Vistaus** [comment]
It would still be great to have this. Even on the biggest zoom factor, the text is almost unreadable for me.

**Comment by strk** [comment]
Using CTRL +/- to make fonts bigger/smaller would be great (it's what I use in Firefox, for all the web-based IM clients)

**Comment by nicodh** [comment]
> It would still be great to have this. Even on the biggest zoom factor, the text is almost unreadable for me.

Can you describe your environment? Show a screenshot and let us know what resolution your device has?

We could add higher zoom level, but only if there is a real need for it...

**Comment by Vistaus** [comment]
Hi Nico,

Environment: KDE Plasma 6 (currently 6.3.5) on CachyOS (Arch Linux), on a 4K screen (3840x2160)
I attached two screenshots, one of Delta Chat and one of Dolphin. This is with Delta Chat on the highest zoom level, but as you can see, text is still very small. System font size is the same everywhere (19/20) and in all other apps it's much bigger, and thus much more readable. Font family also doesn't matter: if I choose e.g. Verdana or Ubuntu, it doesn't make any difference whatsoever with regards to the font size that Delta Chat uses.

![Image](https://github.com/user-attachments/assets/30a1542f-d865-4ea0-8d9e-2314f679da5d)
![Image](https://github.com/user-attachments/assets/4b2ec1fd-ccf3-4b85-a610-5d06be6369a0)

**Comment by Vistaus** [comment]
Yes, in Chromium text size is fine. I can also adjust text size there if I want to, unlike in Delta.

**Comment by nicodh** [comment]
Maybe related to this:
https://github.com/electron/electron/issues/10101

a comment says: "I'm having the same issue with KDE5, so it might be plasma specific. "

**Comment by nicodh** [comment]
And maybe this one, which is newer https://discussion.fedoraproject.org/t/fix-ui-scale-bug-electron-apps-too-small/142940

Sadly I can't reproduce it, so it's hard to find the reason...

**Comment by Vistaus** [comment]
Chat bubbles are a bit oversized and I do lose the contact list with the current window size, but at least I can read the text now, so thanks for this addition! 🙂

**Comment by Vistaus** [comment]
The only real issue is that it doesn't save the zoom level after a restart of Delta Chat.

**Comment by nicodh** [comment]
> The only real issue is that it doesn't save the zoom level after a restart of Delta Chat.

That's a blocker. So I reopen the issue...

**Comment by Vistaus** [comment]
Fix confirmed. Thanks for the quick fix! 🙂

**Comment by Raiden-GH** [comment]
I'm not sure if it's intentional, but in the macOS version of DC-Desktop 1.59.2, you have to press "command" and "*" to zoom in instead of "command" and "+".

**Comment by nicodh** [comment]
You mean the behaviour changed from 1.58.2 to 1.59.2 ?

**Comment by Raiden-GH** [comment]
I didn't know this feature existed before and was referring to this entry in the DC-Desktop 1.59.2 changelog:

`Added: Zoom In/Out with Ctrl +/- #890
`

In my opinion, the combination of "command" and "*" is rather inconvenient, as it requires you to press three keys simultaneously.

**Comment by WofWca** [comment]
> I'm not sure if it's intentional, but in the macOS version of DC-Desktop 1.59.2, you have to press "command" and "*" to zoom in instead of "command" and "+".

Do you know if this is also the case on Signal Desktop? Because we're depending on native Electron behavior for the shortcuts here, and IDK if we can / should revert back to manual keyboard event handling.

**Comment by Raiden-GH** [comment]
Sorry, I don't have Signal, but as far as I know Threema Desktop 2.0 also uses Electron and there it's the combination of "command" and "+".

**Comment by OliverUv** [comment]
Hello!

Hope y'all don't mind me necroing an old issue. Currently I have to strike a balance between zooming in enough that I can read things, and not zooming in too much because all other UI elements also become larger, giving me a very small space for the actual chat.

I am currently unable to get the font size to a comfortable level.

Would be incredibly nice if there was an option somewhere to adjust the font size of just message text and usernames.

EDIT: posted a discussion here: https://support.delta.chat/t/desktop-adjust-font-size-for-messages-and-usernames-only/4383

**Comment by r10s** [comment]
please start a feature proposal at https://support.delta.chat , there the issue can also be weighted against other things to do. reviving stale discussions and closed issues causes too much friction.
